src/Cache/Cache.php

```diff
@@ -0,0 +1,160 @@
+<?php
+
+namespace BBC\ProgrammesPagesService\Cache;
+
+use Psr\Cache\CacheItemInterface;
+use Psr\Cache\CacheItemPoolInterface;
+use InvalidArgumentException;
+
+class Cache implements CacheInterface
+{
+    /** @var CacheItemPoolInterface */
+    private $cachePool;
+
+    /** @var string */
+    private $prefix;
```

Comment:
A prefix, up to the calling application but generally the application version so that changes don't result in invalid objects being unserialised etc.